tools/debug_tools/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+//! Debugging dev tools for Emergence.
+//!
+
+use bevy::{
+    asset::AssetServer,
+    diagnostic::{Diagnostics, FrameTimeDiagnosticsPlugin},
+    prelude::{
+        default, Color, Commands, Component, Plugin, Query, ReflectComponent, Res, Resource,
+        TextBundle, With,
+    }, // Input, KeyCode, // add back to allow for toggling the fps display
```

Comment:
```suggestion
    }, // Input, KeyCode, // TODO: add back to allow for toggling the fps display
```

Reply:
This has been fixed in an upcoming commit